### PR TITLE
Find the current astronomer-providers

### LINKED ISSUE #3513: Add specific example for pulling astronomer providers
### Issue Type

Missing content for an existing feature

### Links to Affected Docs

https://docs.astronomer.io/astro/runtime-image-architecture#provider-package-versioning

### The Issue or Context

User requested a more specific example for finding the astro runtime provider versions .

### Fix or Proposed Change

- In link, add specific command as an example
```
astro dev bash pip freeze | grep astronomer-providers
```
- Add subheading to improve search 

### Additional Notes

https://apache-airflow.slack.com/archives/C03T0AVNA6A/p1708711220828289

### Required Reviewers

_No response_

### PR DESCRIPTION
**Pull request by lzdanski**
Resolves #3513

### CODE REVIEW

File: docs-partials/runtime-image-architecture.mdx [hunk 1]
```diff
@@ -95,6 +95,12 @@ To determine the version of any provider package installed in your current Astro
 docker run --rm <runtime-image> pip freeze | grep <provider>
 ```
 
+For example, to find the version of the current package for Astronomer providers, run the following command:
```
Comment (lzdanski) [inline docs-partials/runtime-image-architecture.mdx:98]: Preview at https://deploy-preview-3514--gallant-neumann-56599d.netlify.app/astro/runtime-image-architecture#provider-package-versioning

Comment (jwitz) [reply docs-partials/runtime-image-architecture.mdx:98]: Can we use a different package example? I think we have plans to no longer support `astronomer-providers`

**Review by jwitz — APPROVED** [review]
LGTM after the example package is changed

File: docs-partials/runtime-image-architecture.mdx [hunk 2]
```diff
@@ -95,6 +95,12 @@ To determine the version of any provider package installed in your current Astro
...
+
+```
+astro dev bash pip freeze | grep astronomer-providers
```
Comment (lzdanski) [inline docs-partials/runtime-image-architecture.mdx:101]:
```suggestion
astro dev bash pip freeze | grep apache-airflow-providers-openlineage
```

File: docs-partials/runtime-image-architecture.mdx (hunk 1 above)
Comment (lzdanski) [inline docs-partials/runtime-image-architecture.mdx:98]:
```suggestion
For example, to find the version of the current package for OpenLineage, run the following command:
```

File: docs-partials/runtime-image-architecture.mdx [hunk 3]
```diff
@@ -95,6 +95,12 @@ To determine the version of any provider package installed in your current Astro
 docker run --rm <runtime-image> pip freeze | grep <provider>
 ```
 
+For example, to find the version of the current package for OpenLineage, run the following command:
+
+```
+astro dev bash pip freeze | grep apache-airflow-providers-openlineage
```
Comment (lzdanski) [inline docs-partials/runtime-image-architecture.mdx:101]:
```suggestion
For example, to find the version of the current package for Astronomer providers, run the following command:

```
astro dev bash pip freeze | grep astronomer-providers
```

Comment (lzdanski) [inline docs-partials/runtime-image-architecture.mdx:101]:
```suggestion
astro dev bash pip freeze | grep astronomer-providers
```

File: docs-partials/runtime-image-architecture.mdx [hunk 4]
```diff
@@ -95,6 +95,12 @@ To determine the version of any provider package installed in your current Astro
 docker run --rm <runtime-image> pip freeze | grep <provider>
 ```
 
+For example, to find the version of the current package for OpenLineage, run the following command:
```
Comment (lzdanski) [inline docs-partials/runtime-image-architecture.mdx:98]:
```suggestion
For example, to find the version of the current package for Astronomer providers, run the following command:
```

**Pull request merged**